### PR TITLE
Compatibility fixes

### PR DESCRIPTION
**Pull request by mponomar**
- Don't use datetime prefix for log messages
- Fix bug where we latch an invalid timezone
- fastinit should trigger auto-analyze

### CODE REVIEW

File: schemachange/sc_fastinit_table.c [hunk 1]
```diff
@@ -131,10 +131,21 @@ int do_fastinit(struct ireq *iq, struct schema_change_type *s, tran_type *tran)
...
 {
     int rc = 0;
+    int is_sqlite =  strncmp((s->table), "sqlite_stat", sizeof("sqlite_stat") - 1) == 0;
+
     rc = finalize_alter_table(iq, s, tran);
+
+    if(!s->drop_table && !is_sqlite) {
+        //triger analyze of table -- when fastinit
+        char * tblname = strdup((char*) s->table); //will be freed in auto_analyze_table()
+        autoanalyze_after_fastinit(tblname);
```
Comment (ldennis) [inline schemachange/sc_fastinit_table.c]: I prefer to move this analyze logic to bb-plugins/schema_change . For DDLs (especially transactional DDLs), running finalize_fastinit_table doesnt necessarily mean the transaction is going to commit. I think it is better to move analyze code to the point after the schema change finishes.

Comment (mponomar) [reply schemachange/sc_fastinit_table.c]: You got it.

File: bb/logmsg.c [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 static loglvl level = LOGMSG_WARN;
 static int do_syslog = 0;
-static int do_time = 1;
+static int do_time = 0;
```
Comment (ldennis) [inline bb/logmsg.c]: No timestamp by default? What about adding "logmsg notimestamp" to legacy default?

Comment (mponomar) [reply bb/logmsg.c]: You find those helpful?  The default right now depends on whether the database runs with a tty.  Can certainly keep the default and disable with legacy defaults.

Comment (ldennis) [reply bb/logmsg.c]: I kinda like the timestamps personally, but maybe I only want them for debugging purposes. So I guess actually it doesnt really matter

**Pull request merged**